Dialog/TilesetEditDialog.cpp

```diff
@@ -107,6 +107,29 @@ void TilesetEditDialog::SetSpinboxesTile8x8sInfo(LevelComponents::Tile8x8* tile8
     checkBoxVFlip->setChecked(tile8->GetFlipY());
 }
 
+/// <summary>
+/// Delete a foreground TIle8x8 from the Tileset and keep all the Tile16 using the correct tile8x8s
+/// </summary>
+/// <param name="tile8x8id">
+/// id of the Tile8x8 which are going to be deleted, the first foreground tile8x8 is indexed 0x41.
+/// </param>
+void TilesetEditDialog::DeleteFGTile8x8(int tile8x8id)
+{
+    tilesetEditParams->newTileset->DelTile8x8(tile8x8id);
+
+    // UI update
+    ReRenderTile16Map();
+    ReRenderTile8x8Map(SelectedPaletteId);
+    if(tile8x8id <= (0x40 + tilesetEditParams->newTileset->GetfgGFXlen() / 32))
```

Comment:
Is this a check for animated tiles? If so, why is there a difference where we subtract 1? Just trying to understand it

Reply:
there are 0x40 animated tiles indexed from 0 to 0x3F, and there is another blanktile indexed 0x40.